src/extension/options-page/Home.tsx

```diff
@@ -0,0 +1,230 @@
+import React from 'react'
+
+import { makeStyles, createStyles } from '@material-ui/styles'
+import {
+    Theme,
+    Typography,
+    Card,
+    Divider,
+    List,
+    ListItem,
+    ListItemText,
+    ListItemSecondaryAction,
+    Container,
+} from '@material-ui/core'
+import { Link, useHistory, useRouteMatch, Redirect } from 'react-router-dom'
+import { useMyPersonas } from '../../components/DataSource/useActivatedUI'
+import Services from '../service'
+
+import { geti18nString } from '../../utils/i18n'
+
+import { DialogRouter } from './DashboardDialogs/DialogBase'
+
+import PersonaCard from './DashboardComponents/PersonaCard'
+import { DatabaseRestoreDialog } from './DashboardDialogs/Database'
+import { PersonaCreateDialog, PersonaCreatedDialog, PersonaImportDialog } from './DashboardDialogs/Persona'
+import ActionButton from './DashboardComponents/ActionButton'
+import FooterLine from './DashboardComponents/FooterLine'
+
+const useStyles = makeStyles((theme: Theme) =>
+    createStyles({
+        sections: {
+            width: '100%',
+            marginTop: theme.spacing(2),
+        },
+        title: {
+            marginTop: theme.spacing(3),
+            marginBottom: theme.spacing(1),
+        },
+        button: {
+            width: 120,
+        },
+        secondaryAction: {
+            paddingRight: 120,
+        },
+        actionButtons: {
+            margin: theme.spacing(2),
+        },
+        loaderWrapper: {
+            position: 'relative',
+            '&:not(:last-child)': {
+                marginBottom: theme.spacing(2),
+            },
+        },
+        loader: {
+            width: '100%',
+            bottom: 0,
+            position: 'absolute',
+        },
+        identity: {
+            '&:not(:first-child)': {
+                marginTop: theme.spacing(2),
+            },
+        },
+    }),
+)
+
+const ListItemWithAction = (props: any) => {
+    const { secondaryAction } = useStyles()
+    return <ListItem classes={{ secondaryAction }} {...props} />
```

Comment:
`{...props} classes={{ ...props.classes, secondaryAction: classnames(secondaryAction, classes.secondaryAction) }}`

Reply:
this component should not be styled